src/Core/Checkout/Document/Service/DocumentGenerator.php

```diff
@@ -161,6 +176,7 @@ public function generate(string $documentType, array $operations, Context $conte
                     'config' => $document->getConfig(),
                     'deepLinkCode' => $deepLinkCode,
                     'referencedDocumentId' => $operation->getReferencedDocumentId(),
+                    'documentMediaFileIds' => [$mediaId, $mediaIdForHtmlA11y],
```

Comment:
```suggestion
                    'documentMediaFileIds' => array_filter([$mediaId, $mediaIdForHtmlA11y]),
```

Reply:
instead of having documentMediaFileIds, can we insert 2 records with 2 different types? so we don't need to add a new field?

Reply:
Thanks for idea, I will try it 💯